tests/unit/requirements.txt

```
@@ -0,0 +1 @@
+hvac ; python_version > '2.6'
```

Shouldn't this be:
```suggestion
hvac ; python_version >= '2.7'
```
Or does it support Python 2.6.1? (Maybe the syntax excludes 2.6.1 as well, I don't know the syntax well enough to determine that, but it's definitely clearer to everyone if you use `>=` :) )